### PR TITLE
Scalability demo

### LINKED ISSUE #856: Local cluster tutorial demonstrating scalability
Plan:

1. Create 3 nodes
2. Lower max range size in default zone config, e.g.:

   ```
   cat default_update.yaml
   range_min_bytes: 262144
   range_max_bytes: 262144

   cockroach zone set .default -f default_update.yaml
   ```
3. Use [`block_writer`](https://github.com/cockroachdb/examples-go/tree/master/block_writer) example to generate load. (Alternative: use @knz SQL [here](https://github.com/cockroachdb/cockroach/issues/11208).)
4. Watch replica count increase in admin ui.
5. When nodes are filling up, add 2 more nodes.
6. Watch rebalancing in admin ui.

### PR DESCRIPTION
**Pull request by jseldess**
This PR adds a new tutorial demonstrating scaling and auto-rebalancing with a local cluster.

Fixes #856

---
This change is [Reviewable](https://reviewable.io/reviews/cockroachdb/docs/899)

### CODE REVIEW

**Comment by jseldess** [comment]
@spencerkimball, PTAL. For now, I'm just assuming that users have Go, or are willing to install it, unless we can make mac and linux binaries easily available (brief talk with @tamird made that sound not likely).

@tamird, please remind me again of the commands you'd prefer over `wget`. I tried `go install github.com/cockroachdb/examples-go/block_writer` but wasn't successful...

**Comment by tamird** [comment]
Ah, it's `go get`, not `go install`.

**Comment by jseldess** [comment]
Thanks, @tamird. Updated accordingly.

File: demo-scalability.md [hunk 1]
```diff
@@ -1,8 +1,176 @@
 ---
 title: Scalability
-summary: 
+summary: Use a local cluster to explore how CockroachDB automatically rebalances data as you scale.
 toc: false
-feedback: false
 ---
 
-Coming soon: Use a local cluster to explore how easy it is to scale CockroachDB with your data.
+This page walks you through a simple demonstration of how CockroachDB automatically rebalances data as you scale. Starting with a 3-node local cluster, you'll lower the maximum size for a single range, the unit of data that is replicated in CockorachDB. You'll then download and connect the `block_writer` example program, which will continuously insert a large amount of data into your cluster, and watch the replica count quickly increase as ranges split. You'll then add 2 more nodes and watch how CockroachDB automatically rebalances replicas to efficiently use all available capacity.
```
Comment (spencerkimball) [inline demo-scalability.md]: s/which will continuously insert a large amount of data/which continuously inserts data/

This sounds more precise:

s/efficiently use all available/spread the load across the available/

"efficiently" is kind of marketing-y

File: demo-scalability.md [hunk 2]
```diff
@@ -1,8 +1,176 @@
...
+range_max_bytes: 262144
+gc:
+  ttlseconds: 86400
+num_replicas: 3
+constraints: []
+~~~
+
+## Step 3. Download and run the `block_writer` program
+
+CockroachDB provides a number of [example programs in Go](https://github.com/cockroachdb/examples-go) for simulating client workloads. The program you'll use for this demonstration is called [`block_writer`](https://github.com/cockroachdb/examples-go/tree/master/block_writer). It will simulate multiple clients quickly and continuously inserting large amounts of data into the cluster.
```
Comment (spencerkimball) [inline demo-scalability.md]: s/clients quickly and continuously inserting/clients inserting/

**Comment by jseldess** [comment]
Done. Thanks, @spencerkimball.

**Pull request merged**